borsh/src/schema.rs

```diff
@@ -903,3 +903,16 @@ impl BorshSchema for std::net::IpAddr {
         id_addr_std_derive_impl::IpAddr::declaration()
     }
 }
+
+/// Adds schema to `bytes::Bytes`.
+/// Conservatively just uses `[u8]` schema, so that any consumer already handles it.
+/// while in Rust code we can be more optimal as needed.
+#[cfg(feature = "bytes")]
+impl BorshSchema for bytes::Bytes {
+    fn add_definitions_recursively(definitions: &mut BTreeMap<Declaration, Definition>) {
+        <[u8]>::add_definitions_recursively(definitions);
+    }
+    fn declaration() -> Declaration {
+        <[u8]>::declaration()
```

Comment:
1. i would change `declaration()` to return `"bytes::Bytes".to_owned()`
2. add similar implementation for `bytes::BytesMut`
3. add a 
```rust
#[cfg(feature = "bytes")]
mod test_bytes;
```
test to test both somewhere here https://github.com/near/borsh-rs/blob/master/borsh/tests/tests.rs#L105, 
using `insta::assert_snapshot!` https://github.com/near/borsh-rs/blob/master/borsh/tests/schema/test_ip_addr.rs#L10

Reply:
1. does not sounds well imho. Because array and slice map to vec, why not bytes?